components/EditProfilePage/EditProfilePage.tsx

```diff
@@ -25,9 +26,15 @@ import {
 import { TestimoniesTab } from "./TestimoniesTab"
 import { useFlags } from "components/featureFlags"
 import { PendingUpgradeBanner } from "components/PendingUpgradeBanner"
-import { TabContext } from "components/shared/ProfileTabsContext"
 
-export function EditProfile() {
+const tabTitles = ["about-you", "testimonies", "following"] as const
+export type TabTitles = (typeof tabTitles)[number]
+
+export default function EditProfile({
+  tabTitles = "about-you"
```

Comment:
np: Should this be singular, i.e. tabTitle? So we don't confuse it with the const declared above?

Reply:
makes sense: done